app/controllers/Mod.scala

```diff
@@ -427,8 +427,8 @@ final class Mod(
               modApi.setPermissions(me, user.username, Permission(permissions)) >> {
                 newPermissions(Permission.Coach) ?? env.security.automaticEmail.onBecomeCoach(user)
               } >> {
-                Permission(permissions).exists(_ is Permission.SeeReport) ?? (env.plan.api
-                  .setLifetime(user) >> !user.isPatron ?? env.activity.write.plan(user.username, 1))
+                Permission(permissions).exists(_ is Permission.SeeReport) ?? env.plan.api
```

Comment:
Wouldn’t that change revert #8744?

Reply:
I guess so but showing "supported for 1 month" also seems weird and they could also already have regular patron months. Maybe adding a separate "is now a lifetime Patron" message would be best but not sure whether this is common enough that it's worth adding a translation string for that, especially since it will only show up once for a month or so and then never again.

I just made it emit a "started supporting" message now if they haven't been a patron before.